tools/rna_tools/rbpbench/rbpbench.xml

```diff
@@ -0,0 +1,1040 @@
+<tool id="rbpbench" name="RBPBench" version="0.1.0+galaxy0" python_template_version="3.5" profile="21.05">
+
+    <description>- Evaluate CLIP-seq and other genomic region data using a comprehensive collection of RBP binding motifs</description>
+    <requirements>
+        <requirement type="package" version="0.6">rbpbench</requirement>
```

Comment:
```suggestion
        <requirement type="package" version="0.6">rbpbench</requirement>
        <requirement type="package" version="5.3">meme</requirement>
```

Reply:
How can I see the test output here? Cannot find it ... I could add meme >=5.0 in the bioconda recipe (https://github.com/bioconda/bioconda-recipes/blob/143a88e35c4f8cce93c4c7d335514e3afb4ce42f/recipes/rbpbench/meta.yaml#L28C9-L28C9). If I only update this, I need to increment "number:" to 1 right? (still keeping version 0.6)

Reply:
Ok, I did some update: https://github.com/bioconda/bioconda-recipes/pull/44566 :D

Reply:
> How can I see the test output here? Cannot find it ... I could add meme >=5.0 in the bioconda recipe (https://github.com/bioconda/bioconda-recipes/blob/143a88e35c4f8cce93c4c7d335514e3afb4ce42f/recipes/rbpbench/meta.yaml#L28C9-L28C9). If I only update this, I need to increment "number:" to 1 right? (still keeping version 0.6)

Yes, increase the build number.

Reply:
@michauhl  wait, I think we have to update it again. Does it work with versions above 5.5.4?